pkg/labels/matcher_test.go

```diff
@@ -87,3 +87,34 @@ func TestMatcher(t *testing.T) {
 		}
 	}
 }
+
+func TestInverse(t *testing.T) {
+	tests := []struct {
+		matcher  *Matcher
+		expected *Matcher
+	}{
+		{
+			matcher:  &Matcher{Type: MatchEqual, Name: "name1", Value: "value1"},
+			expected: &Matcher{Type: MatchNotEqual, Name: "name1", Value: "value1"},
+		},
+		{
+			matcher:  &Matcher{Type: MatchNotEqual, Name: "name2", Value: "value2"},
+			expected: &Matcher{Type: MatchEqual, Name: "name2", Value: "value2"},
+		},
+		{
+			matcher:  &Matcher{Type: MatchRegexp, Name: "name3", Value: "value3"},
+			expected: &Matcher{Type: MatchNotRegexp, Name: "name3", Value: "value3"},
+		},
+		{
+			matcher:  &Matcher{Type: MatchNotRegexp, Name: "name4", Value: "value4"},
+			expected: &Matcher{Type: MatchRegexp, Name: "name4", Value: "value4"},
+		},
+	}
+
+	for _, test := range tests {
+		result, _ := test.matcher.Inverse()
+		if test.expected.Type != result.Type {
```

Comment:
We try to use `testutil.Assert` rather than an `if` and `t.Fatal`. If you like you could update the other test in here as well.